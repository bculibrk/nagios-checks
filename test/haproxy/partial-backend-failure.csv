# pxname,svname,qcur,qmax,scur,smax,slim,stot,bin,bout,dreq,dresp,ereq,econ,eresp,wretr,wredis,status,weight,act,bck,chkfail,chkdown,lastchg,downtime,qlimit,pid,iid,sid,throttle,lbtot,tracked,type,rate,rate_lim,rate_max,check_status,check_code,check_duration,hrsp_1xx,hrsp_2xx,hrsp_3xx,hrsp_4xx,hrsp_5xx,hrsp_other,hanafail,req_rate,req_rate_max,req_tot,cli_abrt,srv_abrt,comp_in,comp_out,comp_byp,comp_rsp,lastsess,last_chk,last_agt,qtime,ctime,rtime,ttime,
ubuntuusers-static,she,0,0,0,0,512,0,0,0,,0,,0,0,0,0,UP,1,1,0,19,5,559046,168,,1,18,1,,0,,2,0,,0,L4OK,,6,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-static,kanu,0,0,0,0,512,0,0,0,,0,,0,0,0,0,DOWN,1,1,0,1,1,4857267,4857267,,1,18,2,,0,,2,0,,0,L4TOUT,,2001,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-static,BACKEND,0,0,0,1,820,761,183203,108701,0,0,,0,0,0,0,UP,1,1,0,,5,559046,168,,1,18,0,,0,,1,0,,2,,,,0,0,761,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
ubuntuusers-media,she,0,0,0,0,512,0,0,0,,0,,0,0,0,0,UP,1,1,0,18,4,559046,163,,1,20,1,,0,,2,0,,0,L4OK,,6,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-media,kanu,0,0,0,0,512,0,0,0,,0,,0,0,0,0,DOWN,1,1,0,1,1,4857266,4857266,,1,20,2,,0,,2,0,,0,L4TOUT,,2001,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-media,BACKEND,0,0,0,0,820,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,4,559046,163,,1,20,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
ubuntuusers-legacy,she,0,0,0,0,512,0,0,0,,0,,0,0,0,0,UP,1,1,0,19,5,559046,167,,1,21,1,,0,,2,0,,0,L4OK,,6,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-legacy,kanu,0,0,0,0,512,0,0,0,,0,,0,0,0,0,DOWN,1,1,0,1,1,4857266,4857266,,1,21,2,,0,,2,0,,0,L4TOUT,,2001,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-legacy,BACKEND,0,0,0,0,820,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,5,559046,167,,1,21,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
