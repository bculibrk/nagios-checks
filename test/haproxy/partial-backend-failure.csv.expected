HAPROXY WARN: ubuntuusers-static kanu (act) DOWN	sess=0/512(0%) smax=0; ubuntuusers-media kanu (act) DOWN	sess=0/512(0%) smax=0; ubuntuusers-legacy kanu (act) DOWN	sess=0/512(0%) smax=0
ubuntuusers-legacy BACKEND (act) UP	sess=0/820(0%) smax=0
ubuntuusers-legacy kanu (act) DOWN	sess=0/512(0%) smax=0
ubuntuusers-legacy she (act) UP	sess=0/512(0%) smax=0
ubuntuusers-media BACKEND (act) UP	sess=0/820(0%) smax=0
ubuntuusers-media kanu (act) DOWN	sess=0/512(0%) smax=0
ubuntuusers-media she (act) UP	sess=0/512(0%) smax=0
ubuntuusers-static BACKEND (act) UP	sess=0/820(0%) smax=1
ubuntuusers-static kanu (act) DOWN	sess=0/512(0%) smax=0
ubuntuusers-static she (act) UP	sess=0/512(0%) smax=0
